.github/workflows/testing-alpine.yml

```diff
@@ -0,0 +1,35 @@
+name: build and test on AlpineLinux
+
+on:
+  push:
+    branches: [ master, devel ]
+  pull_request:
+    branches: [ master, devel ]
+
+jobs:
+  testing:
+    strategy:
+      fail-fast: false
+      matrix:
+        tag: ['3.20','3.19','3.18','3.17']
+        build-type: [Debug,Release]
```

Comment:
Do you think it is necessary to build both release and debug? I'm not very proficient with cmake, so I might not be using it properly, but the thing is, currently the target `pegof` builds optimized version (i.e. release) and `pegof_test` builds debug version.

Running it with `CMAKE_BUILD_TYPE=Release` in current setup results in commands like `-O3 -DNDEBUG ... --coverage -g -O0 -fsanitize=address -fsanitize=undefined` for `pegof_test`. It would need to more changes to support this.

Reply:
In Release mode, do not build and run tests.

Reply:
This will probably need a few more changes, but I'd leave that for a separate PR. For now, I think building only Debug is fine, since we don't publish the binaries anyway.